portfolio/src/main/webapp/index.html

```diff
@@ -70,35 +74,38 @@ <h3>Projects I've worked in </h3>
         </div>
         <!-- Gallery Container -->
         <div id="my_gallery">
-          <h2>My Gallery</h2>
+          <h2 class="page_headers">My Gallery</h2>
           <div id="paragraph_container">
             <p class="justify_text">
               One hobby that I ocassionally have is taking pictures. 
               Even thought I'm not much of a photographer, 
               I really like to capture the moment of any special time that I'm having in my life,
               just for when I see that picture I can remember the feeling that I was having in that moment.
             </p>
-            <button class = "random_picture_button" onclick="getRandomImage()">Click to get a random image</button>
+            <button class = "random_picture_button" onclick="getRandomImage()">Click to start the random image cycler</button>
```

Comment:
```suggestion
            <button class = "random_picture_button" onclick="onClickImageCyclerButton()">Click to start the random image cycler</button>
```

Since the behavior of this button has changed (it no longer retrieves a single random image, but instead cycles through all of them randomly), I would change the name of the above function to more accurately reflect what is happening.

Reply:
Sounds good Aki, thank you!